### ISSUE TITLE
[Request] X/Y Settings

### ISSUE THREAD

**Issue by TheExceed** [Annosz/UIInfoSuite2#93]
Would be great to have adjustable settings for moving the window with the timer on plants/machines.
I currently use combined machines which also has a window and would love to be able to move them up just a tad so they don't look weird when only one shows up.

**Comment by Annosz** [comment]
Hi! I'm not familiar with that mod, can you please send a picture from your game with the wrong window arrangements?

**Comment by TheExceed** [comment]
This is after I tweaked the other mods settings because they do offer these settings.
https://prnt.sc/1y9n6ej

However, when there is no info from UI Info Suite, it looks very off due to the spacing.
https://prnt.sc/1y9n9o8

By default, both mods are essentially at the same location unless settings are changed for [Combine Machines.](https://www.nexusmods.com/stardewvalley/mods/7411)

**Comment by OHchin** [comment]
hey i have a problem because ui info suite2 dont have a dll file 
![image](https://user-images.githubusercontent.com/77050973/142769689-ed5c1164-c65e-4a23-bbb4-9eee1060b599.png)

**Comment by D3vilEye** [comment]
Yeah, please! Im using the Magic Mod by spacechase0 and patochild and the Ui from that mod overlap the UiInfoSuite2 one.
